src/Domain/Models/User.cs

```diff
@@ -9,10 +10,15 @@
 namespace Domain.Models
 {
     public class User : IdentityUser
-    {
+    {
         [Required]
-        public string Name { get; set; }
+        public string Name { get; set; }

```

Comment:
Either add a property to support both `FirstName` and `LastName` or rename to `FullName`